src/design/JSONschemaForm/styles.css

```diff
@@ -0,0 +1,12 @@
+.array-item {
```

Comment:
What does this file do exactly? Is there any chance we can get rid of it and use tailwind?

Reply:
This css defines the size of toolbox which are "remove", "move up", "move down" buttons
The auto-generated code for the form has the Bootstrap classes like **col-xs-9** that actually has no meaning in our code base
So, we should define those classes
We have no way to insert tailwind classes as they are generated by the Form library

Reply:
I wonder if we could still get it to work with tailwind when we have access to some parent.

```html
<div className="[&_.some-class-very-deep-in-the-dom]:border">
  <div>
    <div>
      <div className="some-class-very-deep-in-the-dom">
        I have a border
      </div>
    </div>
  </div>
</div>
``` 

I agree that this is not very intuitive, but I would love to have as little custom CSS files as possible. I also had an issue with playwright the other day when it just could not run when it parsed `import "./somecss.css";` somewhere in the code (which I found super weird because it works in `src/App.tsx`)

Reply:
Removed the usage of the style.css
So this is resolved